manifests/scenario/base.pp

```diff
@@ -4,123 +4,40 @@
-#   Add a ``host`` entry for the Puppet server to the catalog
-#
-#   * This has no effect if the ``$server_facts`` Hash is not populated
-#
-# @param use_sudoers_aliases
-#   If true, enable simp site sudoers aliases
-#
-# @param runlevel
-#   The default runlevel to which the system should be set
-#
-# @param restrict_max_logins
-#   Enable restrictions of the number of simultaneous logins a user may have
-#
-#   * Has no effect if ``$pam`` is ``false``
-#
-# @param manage_ctrl_alt_del
-#   Include the ``simp::ctrl_alt_del`` class, which, by default, disables the
-#   use of ctrl_alt_del and logs all instances of the event.
-#
-# @param manage_root_metadata
-#   Include the ``simp::root_user`` class, which manages resources related to
-#   the ``root`` user
-#
-# @param manage_root_perms
-#   Ensure that ``/root`` has restricted permissions and proper SELinux
-#   contexts
-#
-# @param manage_rc_local
-#   Include the ``simp::rc_local`` class
-#
-#   * This **disables** rc.local by default but you may also use it to set
-#     custom content
-#
-# @param pam
-#   Enable management of PAM resources via SIMP modules
-#
-# @param sssd
-#   Enable management of SSSD resources via SIMP modules
-#
-# @param ldap
-#   Enable management of LDAP resources via SIMP modules
-#
-# @param stock_sssd
-#   Add a default setup that will successfully connect to the SIMP LDAP server,
-#   if enabled, and will otherwise provide a functional SSSD stack for the
-#   system
-#
-#   * Has no effect if ``$sssd`` is ``false``
-#
 # @author Trevor Vaughan <tvaughan@onyxpoint.com>
-#
-class simp::scenario::base (
-  Variant[Boolean,Enum['remote']] $mail_server                = $::simp::mail_server,
-  Variant[Boolean,Simplib::Host]  $rsync_stunnel              = $::simp::rsync_stunnel,
-  Boolean                         $use_ssh_global_known_hosts = $::simp::use_ssh_global_known_hosts,
-  Boolean                         $puppet_server_hosts_entry  = $::simp::puppet_server_hosts_entry,
-  Boolean                         $use_sudoers_aliases        = $::simp::use_sudoers_aliases,
-  Simp::Runlevel                  $runlevel                   = $::simp::runlevel,
-  Boolean                         $restrict_max_logins        = $::simp::restrict_max_logins,
-  Boolean                         $manage_ctrl_alt_del        = $::simp::manage_ctrl_alt_del,
-  Boolean                         $manage_root_metadata       = $::simp::manage_root_metadata,
-  Boolean                         $manage_root_perms          = $::simp::manage_root_perms,
-  Boolean                         $manage_rc_local            = $::simp::manage_rc_local,
-  Boolean                         $pam                        = $::simp::pam,
-  Boolean                         $ldap                       = $::simp::ldap,
-  Boolean                         $sssd                       = $::simp::sssd,
-  Boolean                         $stock_sssd                 = $::simp::stock_sssd
-) inherits ::simp {
```

Comment:
@heliocentric those changes were at my request and they were short-sighted as I forgot about API backwards compatibility in favor of "pretty code."  Thanks for catching that, and I agree that they should be added back in until we are ready to do a major version bump.